cmd/buildah/pull.go

```diff
@@ -100,6 +105,11 @@ func pullCmd(c *cobra.Command, args []string, iopts pullResults) error {
 
 	id, err := buildah.Pull(getContext(), args[0], options)
 	if err != nil {
+		cause := errors.Cause(err)
+		_, isNetErr := cause.(net.Error)
+		if isNetErr || cause == errTlsUnexpectedMsg {
```

Comment:
Can the `cause == errTlsUnexpectedMsg` test ever succeed?

Reply:
I can't duplicate this `tls: unexpected message` error for now.  Is it an issue has been fixed or an error always with fedora registry?